gift_card.py

```diff
@@ -348,3 +352,129 @@ def parse(cls, report, records, data, localcontext):
         return super(GiftCardReport, cls).parse(
             report, records, data, localcontext
         )
+
+
+class GiftCardRedeemStart(ModelView):
+    "Gift Card Redeem Start View"
+    __name__ = 'gift_card.redeem.start'
+
```

Comment:
The user must also be given the option to enter the amount. Also check if the amount to be redeemed is less than the available amount in GC